docs/LoaderDebugging.md

```diff
@@ -0,0 +1,322 @@
+DRIVER:       /usr/local/share/vulkan/icd.d
+DRIVER:       /usr/share/vulkan/icd.d
+DRIVER:    Found the following files:
+DRIVER:       /usr/share/vulkan/icd.d/intel_icd.x86_64.json
+DRIVER:       /usr/share/vulkan/icd.d/lvp_icd.x86_64.json
+DRIVER:       /usr/share/vulkan/icd.d/radeon_icd.x86_64.json
+DRIVER:       /usr/share/vulkan/icd.d/lvp_icd.i686.json
+DRIVER:       /usr/share/vulkan/icd.d/radeon_icd.i686.json
+DRIVER:       /usr/share/vulkan/icd.d/intel_icd.i686.json
+DRIVER:       /usr/share/vulkan/icd.d/nvidia_icd.json
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/intel_icd.x86_64.json, version "1.0.0"
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/lvp_icd.x86_64.json, version "1.0.0"
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/radeon_icd.x86_64.json, version "1.0.0"
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/lvp_icd.i686.json, version "1.0.0"
+DRIVER: Requested driver /usr/lib/libvulkan_lvp.so was wrong bit-type. Ignoring this JSON
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/radeon_icd.i686.json, version "1.0.0"
+DRIVER: Requested driver /usr/lib/libvulkan_radeon.so was wrong bit-type. Ignoring this JSON
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/intel_icd.i686.json, version "1.0.0"
+DRIVER: Requested driver /usr/lib/libvulkan_intel.so was wrong bit-type. Ignoring this JSON
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/nvidia_icd.json, version "1.0.0"
+```
+
+Then when the application selects the device to use, you will see the Vulkan
+device call chain reported in the following way (NOTE: additional spaces have
+been removed from the output for easier reading):
+
+```
+DRIVER: vkCreateDevice layer callstack setup to:
+DRIVER:    <Application>
+DRIVER:      ||
+DRIVER:    <Loader>
+DRIVER:      ||
+DRIVER:    <Device>
+DRIVER:        Using "Intel(R) UHD Graphics 630 (CFL GT2)" with driver: "/usr/lib64/libvulkan_intel.so"
+```
+
+
+### Selectively Enable Specific Drivers
+
+You can now use the filtering environment variables
+(`VK_LOADER_DRIVERS_SELECT` and `VK_LOADER_DRIVERS_DISABLE`) to selectively
+select or disable various drivers.
+Remember, that to disable drivers, you must use the name of the Driver JSON to
+disable since the drivers do not reveal a name until much later in the Vulkan
+initialization process.
+
+So to disable all drivers except Nvidia you could do the following:
+
+```
+set VK_LOADER_DRIVERS_DISABLE=*
+set VK_LOADER_DRIVERS_SELECT=*nvidia*
+```
+
+The loader outputs messages like the following when the environment variables
+are used:
+
+```
+WARNING | DRIVER: Driver "intel_icd.x86_64.json" ignored because not selected by env var 'VK_LOADER_DRIVERS_SELECT'
+WARNING | DRIVER: Driver "radeon_icd.x86_64.json" ignored because it was disabled by env var 'VK_LOADER_DRIVERS_DISABLE'
+```
+
+These can be used to make sure that the appropriate drivers are enabled/disabled
+properly.
```

Comment:
`properly`? What would improperly enabled/disabled drivers be? ;)

I would just remove the sentence.

Reply:
👍🏽